### PR TITLE
RELATED: RAIL-1585 improve XSS prevention

### PR DESCRIPTION
**Pull request by no23reason**
[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2284
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2021

### CODE REVIEW

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1022.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1474/ : SUCCESS in 15m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4925/ : SUCCESS in 1m 03s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3887/ : FAILURE in 25m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4924/ : SUCCESS in 24s

**Comment by no23reason** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1022.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3889/ : SUCCESS in 23m 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4926/ : SUCCESS in 19s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1475/ : SUCCESS in 9m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4927/ : SUCCESS in 8s

**Comment by no23reason** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2124/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1476/ : SUCCESS in 10m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4928/ : SUCCESS in 15s

File: src/components/core/pivotTable/tests/agGridUtils.spec.ts [hunk 1]
```diff
@@ -75,22 +73,6 @@ describe("getGridIndex", () => {
     );
 });
 
-describe("cellRenderer", () => {
-    it("should escape value", () => {
```
Comment (no23reason) [inline src/components/core/pivotTable/tests/agGridUtils.spec.ts]: This test is now obsolete as the values coming to cellRenderer are sanitized already.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1037/ : FAILURE in 1m 35s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2142/ : SUCCESS in 10m 02s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3918/ : SUCCESS in 6m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1038/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1195/ : FAILURE in 1m 17s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2143/ : SUCCESS in 12m 44s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3919/ : SUCCESS in 19m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1039/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1196/ : SUCCESS in 14m 42s

**Pull request merged**